jsapp/fonts/.gitignore

```diff
@@ -1,11 +0,0 @@
-*.eot
```

Comment:
Watch out for this. We should probably fix the `Dockerfile` so that it doesn't delete this file. Removing this file may have broken your build.